extensions/agroal/deployment/src/test/java/io/quarkus/agroal/test/MultipleDataSourcesConfigTest.java

```diff
@@ -1,37 +1,37 @@
 package io.quarkus.agroal.test;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.fail;
-
 import java.sql.Connection;
 import java.sql.SQLException;
 
 import javax.inject.Inject;
+import javax.inject.Named;
 
+import io.agroal.api.AgroalDataSource;
+import io.agroal.api.configuration.AgroalConnectionPoolConfiguration;
+import io.quarkus.test.QuarkusUnitTest;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
-import io.agroal.api.AgroalDataSource;
-import io.agroal.api.configuration.AgroalConnectionPoolConfiguration;
-import io.quarkus.agroal.DataSource;
-import io.quarkus.test.QuarkusUnitTest;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
 
-@Disabled
 public class MultipleDataSourcesConfigTest {
 
+    //tag::injection[]
     @Inject
+    @Named("default")
     AgroalDataSource defaultDataSource;
 
     @Inject
-    @DataSource("datasource1")
+    @Named("users")
     AgroalDataSource dataSource1;
 
     @Inject
-    @DataSource("datasource2")
+    @Named("inventory")
```

Comment:
Mmmh, my recollection of our discussion was that we would keep this qualifier and the current behavior?

I think we should stick to how it was done. It's not a big change though.